src/pages/tezos/cryptobot/[id].js

```diff
@@ -403,10 +419,21 @@ function BotView({ location }) {
                   setOnSaleError('INSUFFICIENT_AMOUNT');
                 } else {
                   putBotOnSale(bot.tokenId, convertXtzToMutez(salePrice));
+                  setClaimButtonDisabledStatus(true);
                 }
               }}
             >
-              Continue
+              {claimButtonDisabled ? (
+                <div className={`flex justify-center align-center`}>
+                  {getUserBalance.value > 0.5 ? (
+                    <Loader type="ThreeDots" color="#BFDBFE" height={28} />
+                  ) : (
+                    'Yes'
```

Comment:
Is there a state where `Yes` text might show up and if yes, what does it intends to indicate?

Reply:
This happens when we withdraw cryptobot from sale.
![image](https://user-images.githubusercontent.com/49694914/112474659-eb2a6600-8d95-11eb-9282-0cf86b743c4d.png)